veLords/src/reward_pool.cairo

```diff
@@ -19,7 +19,7 @@ mod reward_pool {
     const DAY: u64 = 3600 * 24;
     const WEEK: u64 = DAY * 7;
     const TOKEN_CHECKPOINT_DEADLINE: u64 = DAY;
-    const ITERATION_LIMIT: u32 = 200;
+    const ITERATION_LIMIT: u32 = 10_000;
```

Comment:
@milancermak does the 10k limit not result in an out of gas error. have you tested that?

Reply:
Yes, though not directly - I modified the loops where the `ITERATION_LIMIT` is used to go through the whole 10000 iterations (with ~all the computation that would entail) and then used the `test_reward_pool_claim_pass` to see if it finishes. It did, though of course it eats up a lot of resources, somewhere close to 100M steps if snforge can be trusted 😅

The loops progress by weeks, so I doubt this limit will ever be reached (10 000 weeks is about 191 years).

Reply:
the max steps on mainnet rn is 4m and i think would be increased to 10m in future. anyway, i think snforge can be pretty accurate and 100m steps is way above limit. we could leave it as is but that would mean this pr does not achieve anything as the contract would still theoretically run out gas before the loop is complete. maybe we just take it back to 200 or slightly higher

Reply:
I thought it's 40M and will go to 100M. Anyway, ok, how does 500 sound?

Reply:
Optimistically pushed a commit decreasing the limit to 500 😇